### ISSUE TITLE
Agda reports unsolved constraints without mentioning any

### ISSUE THREAD

**Issue by jespercockx** [agda/agda#4519]
Minimal example:
```agda
_ = λ f x → f x
```
Agda reports the following:
```
Sort _2  [ at ...:2,7-8 ]
_3 : _2  [ at ...:2,7-8 ]
Sort _6  [ at ...:2,9-10 ]
_7 : _6  [ at ...:2,9-10 ]
Sort _8  [ at ...:2,9-16 ]
_9 : _8  [ at ...:2,9-16 ]
_11 : _9  [ at ...:2,13-16 ]

———— Errors ————————————————————————————————————————————————
Unsolved constraints
```
I would Agda to either report some unsolved constraints, or else there shouldn't be the message "Unsolved constraints".

**Comment by andreasabel** [comment]
This seems to be the compromise introduced in #4065: to not print postponed type checking problems, but keep the information that there are unsolved constraints.

Not happy?  
(Then please reopen with a proposal.)

**Comment by jespercockx** [comment]
I'm hitting this issue again on a file that has no unsolved metas but only "unsolved constraints". It is immensely frustrating that Agda gives me absolutely no clue as to what constraint is unsolved, which might help me to find what's causing the problem.

Why does Agda not simply print the unsolved constraints as usual?

**Comment by jespercockx** [comment]
In #4065, @Saizan writes:

> I mean, I wonder if `[5] _6 := (_ : _4) [ℓ] :? Set` contains any useful information, however I suppose we can print such constraints when they get elevated to an error?

Actually this constraint gives me plenty of information, assuming I know the origin of the two metavariables `_4` and `_6`. The fact that Agda currently makes it very hard to trace back the origin of a metavariable should not be an excuse to hide information, but rather to *add more* so the user can make sense of the error. (There is the separate question of how difficult it is to interpret the information in an error, but the information has to be there in the first place.)

**Comment by Saizan** [comment]
@jespercockx I still have no clue how to read that constraint, is it saying that _4 is supposed to be Set? What is `[ℓ]` ?

However my original change regarding #3606 was meant to stop printing the `BlockedConst` solutions, as they are instead inlined into other constraints. 

Now I realize that `UnBlock` constraints can also be related to a `PostponedTypeCheckingProblem`, which I would agree it should be printed out. It should be straightforward to adjust `interestingConstraints`, though it would help to have a clearer printout of these problems.

**Comment by andreasabel** [comment]
2025-10-16: situation unchanged.
